packages/stylelint-plugin/src/rules/no-invalid-use-of-var-tokens-in-equations.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { getParser } from "../util/utils"
+import { noInvalidUseOfVarTokensInEquations } from "./no-invalid-use-of-var-tokens-in-equations"
+
+describe("no-invalid-use-of-var-tokens-in-equations rule", () => {
+  test.each([
```

Comment:
TIL `test.each` - I've been wrapping test cases in `foreach` like a sucker.

Reply:
Yeah this was new for me too!